helm-charts/iqserver/README.md

```diff
@@ -0,0 +1,172 @@
+# Sonatype IQ server
+
+[Sonatype Nexus IQ Server](https://www.sonatype.com/nexus-iq-server)  is everything you need to know to trust your software supply chain. It powers Nexus Firewall, Nexus Lifecycle, and Nexus Auditor.
+
+### Prerequisites
+
+- Kubernetes 1.8+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- Helm 2
+
+### With Open Docker Image
+
+By default, the Chart uses Red Hat's Certified Container. If you want to use the standard docker image, run with `--set iq.imageName=sonatype/nexus-iq-server`.
```

Comment:
is this line still valid with operators?

Reply:
I copied large parts of this readme into the CSV file, where it'll actually be seen in the catalog. As you suggest, I excluded it from the visible CSV file. This readme is just copied during generation, and is not seen.